app/models/event.rb

```diff
@@ -6,6 +6,13 @@ class Event < ActiveRecord::Base
   validates :start_on, presence: true
   validates :end_on, presence: true
   validates :name, presence: true
+  validate :validate_end_on_before_start_on
+
+  def validate_end_on_before_start_on
+    if end_on && start_on
```

Comment:
Fixed everything except for this one. As soon as I remove `if end_on && start_on` I get `undefined method `<' for nil:NilClass`.